ci-operator/step-registry/platform-external/cluster/aws/pre/platform-external-cluster-aws-pre-chain.yaml

```
@@ -0,0 +1,15 @@
+chain:
+  as: platform-external-cluster-aws-pre
+  steps:
+  - chain: platform-external-cluster-pre
+  - ref: platform-external-cluster-aws-install
+  - ref: platform-external-cluster-wait-for-api-bootstrap
+  - ref: platform-external-ccm-conf-aws
+  - ref: platform-external-ccm-deploy
+  - ref: platform-external-cluster-wait-for-ready-control
+  - ref: platform-external-cluster-wait-for-ready-compute
+  - ref: platform-external-cluster-wait-for-complete
```

maybe create a chain to summarize it:
```suggestion
  - chain: platform-external-cluster-wait
```